FluxoDesenvolvimento.md

```diff
@@ -0,0 +1,65 @@
+### ANTES DE COMEÇAR A DESENVOLVER:
+
+1. Clone o repositório
+
+- `git clone git@github.com:tryber/sd-04-challenge-bug-hunting.git`.
```

Comment:
I think it would be better not to use the name "sd-01", "sd-02", "sd-03", "sd-04" etc. I'm saying this because this project is with all classes, so I think it would be better if the name is just `challenge-bug-hunting`. What do you think?

Reply:
Nice one!